front/src/views/NotFoundView/NotFoundView.tsx

```diff
@@ -18,7 +21,9 @@ function NotFoundView() {
   }
 
   // If the requested route was not found redirect to statviz
-  const baseId = user["https://www.boxtribute.com/base_ids"][0];
+
+  const baseId =
+    globalPreferences.selectedBase?.id || user["https://www.boxtribute.com/base_ids"][0];
```

Comment:
maybe using an entry from `src/utils/constants.ts` would be nice here and in App.tsx

Reply:
it is used two times in the code. not yet ready for abstraction in my view since we are touching and refactoring globalPreferences probably soon again.